backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VirtualQueueController.java

```diff
@@ -52,4 +54,9 @@ public GetTokensResponse getTokens(@PathVariable UUID shopId) {
     public GetShopsByShopOwnerResponse getShopsByShopOwner(@PathVariable String shopOwnerId) {
         return virtualQueueService.getShopsByShopOwner(shopOwnerId);
     }
+
+    @DeleteMapping("/token/{tokenId}")
+    public DeleteTokenResponse deleteToken(@PathVariable UUID tokenId, @RequestBody Boolean isLoggedin){
```

Comment:
Just curious, why have we not used the datatype "boolean" and used "Boolean" instead everywhere? (This is not an issue or error. Let's discuss this over call.)

Reply:
https://stackoverflow.com/questions/3728616/boolean-vs-boolean-in-java This is a link I referred to, which says Boolean provides more functionality and a wrapper class for boolean(primitive data type). But still we can have a quick catch up to discuss this.

Reply:
Yup. Our hosts might also guide us here.

Reply:
yes we can talk about it to the hosts.